### PR TITLE
luci-app-natmap: add new package

### PR DESCRIPTION
**Pull request by ysc3839**

### CODE REVIEW

File: applications/luci-app-natmap/root/usr/share/luci/menu.d/luci-app-natmap.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+		"title": "NATMap",
+		"action": {
+			"type": "view",
+			"path": "natmap"
+		},
+		"depends": {
+			"acl": [ "luci-app-natmap" ]
+		}
+	}
+}
```
Comment (feckert) [inline applications/luci-app-natmap/root/usr/share/luci/menu.d/luci-app-natmap.json:12]: missing new line

File: applications/luci-app-natmap/root/usr/share/rpcd/acl.d/luci-app-natmap.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+			"ubus": {
+				"service": [ "list" ]
+			},
+			"uci": ["natmap"]
+		},
+		"write": {
+			"uci": ["natmap"]
+		}
+	}
+}
```
Comment (feckert) [inline applications/luci-app-natmap/root/usr/share/rpcd/acl.d/luci-app-natmap.json:17]: missing new line

**Review by feckert — CHANGES_REQUESTED** [review]
Everything else looks good from my point of view :+1:

**Comment by ysc3839** [comment]
@feckert Hello, did you forget this PR?

File: applications/luci-app-natmap/htdocs/luci-static/resources/view/natmap.js [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+
+		o = s.option(form.Flag, 'enable', _('Enable'));
+		o.editable = true;
+		o.modalonly = false;
+
+		o = s.option(form.ListValue, 'udp_mode', _('Protocol'));
+		o.default = '1';
+		o.value('0', 'TCP');
+		o.value('1', 'UDP');
+		o.textvalue = function(section_id) {
```
Comment (stangri) [inline applications/luci-app-natmap/htdocs/luci-static/resources/view/natmap.js:63]: I'm new to js, would you care to explain to me why this is needed?

Comment (ysc3839) [reply applications/luci-app-natmap/htdocs/luci-static/resources/view/natmap.js:63]: It will show `0` or `1` in grid without this.

**Comment by stangri** [comment]
@hnyman Hannu, I know @feckert said he's very busy, should we merge this, given his blessing for the rest of the code and fixes he requested being implemented?

**Comment by ysc3839** [comment]
@hnyman Can you backport this to 22.03.2? Thanks.

**Pull request merged**